lib/giftbit/base.rb

```
@@ -0,0 +1,79 @@
+require 'active_support'
+
+begin
+  require 'rest-client'
+rescue LoadError
+end
```

Seems bad to be silencing this error, since the gem depends on rest-client